File: src/client/views/navbar_item.ts
// AUTO GENERATED @ 2024-05-17T16:46:47.978Z
export interface NavbarItemProps {
  active: boolean;
  content: string;
  href: string;
}

export function navbar_item(props: NavbarItemProps) {
  return `<a class="p-2 hover:bg-white/10 rounded-full data-[active=true]:bg-white/20 data-[active=true]:pointer-events-none" href="${props.href}" data-active="${props.active}" >${props.content}</a>`;
}

File: src/server/views/navbar_item.rs
// AUTO GENERATED @ 2024-05-17T15:32:35.865Z
pub fn navbar_item(href: &str, active: bool, content: &str) -> String {
    return format!(r#"<a class="p-2 hover:bg-white/10 rounded-full data-[active=true]:bg-white/20 data-[active=true]:pointer-events-none" href="{href}" data-active="{active}" >{content}</a>"#);
}